.github/workflows/run-ank.yml

```diff
@@ -0,0 +1,30 @@
+name: RunAnk
+
+on: pull_request
+
+jobs:
+  type_checking:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+          architecture: x64
+      - name: Validate gradle wrapper
+        uses: gradle/wrapper-validation-action@v1
+      - name: Clean build
+        env:
+          JAVA_OPTS: -Xms512m -Xmx1024m
+        run: |
+          java -version
+          ./gradlew -v
+          ./gradlew clean build
+      - name: Run Ank
+        if: success()
```

Comment:
This `if` is not necessary because a step will be executed if there is a successful execution in the previous one.